Feature/12 hugo conversion (#14)
Cov-ComSec/Cov-ComSec.github.io: content/posts/web_enum_writeup/index.md

> By the Way always check the source code.

Reviewer: ```suggestion
By the way, always check the source code.
```

![](img/404.png)

## .well-known/security.txt

> Have you heard of this directory before if you don't you can get an idea [security.txt](https://en.wikipedia.org/wiki/Security.txt){target=_blank}. Always good to check. 

Reviewer: ```suggestion
Have you heard of this directory before? If you haven't you can read about it [here](https://en.wikipedia.org/wiki/Security.txt){target=_blank}. Always good to check.
```
